File: iterableapi/src/main/java/com/iterable/iterableapi/IterableActionSource.java
package com.iterable.iterableapi;

/**
 * Enum representing the source of the action: push notification, app link, etc.
 */
public enum IterableActionSource {
    /** Push Notification */
    PUSH,

    /** App Link */
    APP_LINK,

    /** In-App Message */
    IN_APP,

    /** Embedded Message */
    EMBEDDED
}

File: iterableapi/src/main/java/com/iterable/iterableapi/IterableActionSource.kt
package com.iterable.iterableapi

/**
 * Enum representing the source of the action: push notification, app link, etc.
 */
enum class IterableActionSource {
    /** Push Notification */
    PUSH,

    /** App Link */
    APP_LINK,

    /** In-App Message */
    IN_APP,

    /** Embedded Message */
    EMBEDDED
}
